controlplane/test/deactivate-org.test.ts

```diff
@@ -0,0 +1,100 @@
+import { setTimeout } from 'node:timers/promises';
+import { EnumStatusCode } from '@wundergraph/cosmo-connect/dist/common/common_pb';
+import { afterAll, beforeAll, describe, expect, test } from 'vitest';
+import { afterAllSetup, beforeAllSetup, genID, TestUser } from '../src/core/test-util.js';
+import { OrganizationRepository } from '../src/core/repositories/OrganizationRepository.js';
+import { createDeleteOrganizationWorker } from '../src/core/workers/DeleteOrganizationWorker.js';
+import { OidcRepository } from '../src/core/repositories/OidcRepository.js';
+import { SetupTest } from './test-util.js';
+
+let dbname = '';
+
+describe('Deactivate Organization', (ctx) => {
```

Comment:
Please add another test to ensure that we consider deactivation with API keys.

Reply:
We use a mock authenticator. Right now we cannot test our authenticator implementation to verify read-only mode.